examples/district/app/snapshot/page.tsx

```diff
@@ -0,0 +1,360 @@
+      value: '1,094',
+    },
+    {
+      text: 'Sex Ratio (Females per 1,000 Males)',
+      value: '1,094',
+    },
+    {
+      text: 'Sex Ratio (Females per 1,000 Males)',
+      value: '1,094',
+    },
+    {
+      text: 'Sex Ratio (Females per 1,000 Males)',
+      value: '1,094',
+    },
+  ];
+
+  const [currentSlide, setCurrentSlide] = React.useState(1);
+  const [childrenLength, setChildrenLength] = React.useState(3);
+
+  const [open, setOpen] = React.useState(false);
+
+  function current(data: number, length: number) {
+    setCurrentSlide(data);
+    setChildrenLength(length);
+  }
+
+  function chunk(items: any, size: any) {
+    const chunks = [];
+    items = [].concat(...items);
+
+    while (items.length) {
+      chunks.push(items.splice(0, size));
+    }
+
+    return chunks;
+  }
+
+  let a: any, b: any, c: any;
+
+  [a, b, c] = chunk(res, 8);
+
+  const indicators = [
+    {
+      value: '1',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '2',
+      helpText:
+        'Amount spent under the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '3',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '4',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '5',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '6',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '7',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '8',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+    {
+      value: '9',
+      helpText:
+        'Total funds available for the scheme in the selected financial year (FY)',
+      text: 'Funds Available',
+    },
+  ];
+
+  return (
+    <div className={styles.Container}>
+      <Text as="h1" variant="heading3xl" fontWeight="bold">
+        Araria
+      </Text>
+      <Box padding="5">
+        <div className="flex justify-between">
+          <Text as="h1" variant="headingXl" fontWeight="bold">
+            <div className=" mb-2">Demographic highlights</div>
+          </Text>
+          <div className="rounded-sm p-2 bg-[color:var(--border-disabled)]">
+            SHOWING 0{currentSlide} / 0{childrenLength}
+          </div>
+        </div>
+        <div className={styles.Carousel}>
```

Comment:
although there is a generic carousel in component library that you are using, you should create a new carousel component in some `/components` directory and use it here